src/main/kotlin/com/terraformation/backend/customer/model/IndividualUser.kt

```diff
@@ -530,6 +539,9 @@ data class IndividualUser(
   override fun canUpdateAccession(accessionId: AccessionId) =
       isManagerOrHigher(parentStore.getFacilityId(accessionId))
 
+  override fun canUpdateApplication(applicationId: ApplicationId) =
```

Comment:
Should TF expert be able to edit applications? I am guessing they can use the review privilege to update comments and feedback.

Reply:
Yes, the review privilege covers admin-only edits. But your comment does highlight that this name is confusing. Will figure out a clearer name.

Reply:
I split this into two permissions, one for updating the boundary (which TF Expert should be able to do) and one for updating the submission status, which covers the "submit" and "restart" operations and which only the org admin can do.

Reply:
Makes sense to me! Thanks!